packages/core/src/components/_index.scss

```diff
@@ -70,7 +63,8 @@ Styleguide components.usage.npm
 Vanilla JS APIs
 
 JS Components are built using React, but that does not limit their usage to just React applications.
-You can render any component in any JavaScript application with `ReactDOM.render`.
+You can render any component in any JavaScript application with `ReactDOM.render`. Think of it like
+using a JQuery plugin.
```

Comment:
if you're going to make this analogy then you're going to have to back it up.

`ReactDOM.render(<Plugin {...options} />, selector)` ~~ `$(selector).plugin(options)`